homeassistant/bootstrap.py

```diff
@@ -98,6 +99,14 @@ def from_config_dict(config: Dict[str, Any],
         _LOGGER.warning("Skipping pip installation of required modules. "
                         "This may cause issues")
 
+    # Kick off loading the registries.
+    # These will be waited for at the end of stage 0.
+    registry_load = asyncio.gather(
+        hass.helpers.device_registry.async_get_registry(),
+        hass.helpers.entity_registry.async_get_registry(),
+        hass.helpers.area_registry.async_get_registry(),
+        loop=hass.loop)
```

Comment:
Since we're in a coroutine we don't need to pass loop explicitly. `asyncio.gather` will use the current loop.